src/Tribe/Views/V2/View.php

```diff
@@ -1624,10 +1624,13 @@ protected function get_breadcrumbs() {
 		if ( $is_featured = tribe_is_truthy( $this->context->get( 'featured', false ) ) ) {
 			$non_featured_link = tribe_events_get_url( [ 'featured' => 0 ] );
 
-			$breadcrumbs[] = [
-				'link'  => $non_featured_link,
-				'label' => tribe_get_event_label_plural(),
-			];
+			if ( empty( $context_tax ) ) {
+				$breadcrumbs[] = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: f208e54ef83c371be7fb8901993bb317</sub>